scripts/qesap/lib/cmds.py

```diff
@@ -126,9 +99,6 @@ def cmd_configure(configure_data, base_project, dryrun):
             return Status(err)
         log.debug("Hana media %s:\n%s", cfg_paths['hana_media_file'], hanamedia_content)
 
-        if 'hana_vars' in configure_data['ansible'] and configure_data['apiver'] >= 2:
```

Comment:
Is this removed on purpose? It seems to be running on the case `apiver >= 2`, so it would run in the case of `apiver=3`, which will be the new condition.

Reply:
yes, was just a log.